protos/perfetto/config/data_source_config.proto

```diff
@@ -108,7 +108,25 @@ message DataSourceConfig {
   // DO NOT SET in consumer as this will be overridden by the service.
   optional uint64 tracing_session_id = 4;
 
-  // Keeep the lower IDs (up to 99) for fields that are *not* specific to
+  enum BufferExhaustedPolicy {
+    // The data source will drop packets when there's no space in the shared
+    // memory buffer.
+    BUFFER_EXHAUSTED_DROP = 0;
```

Comment:
This needs to be BUFFER_EXHAUSTED_UNSPECIFIED = 0 as per https://protobuf.dev/best-practices/dos-donts/#unspecified-enum

Reply:
Good idea, done